src/components/landscape-sidebar/conservation-efforts-widget/conservation-efforts-widget-selectors.js

```diff
@@ -0,0 +1,92 @@
+import { createSelector, createStructuredSelector } from 'reselect';
+import { sumBy } from 'lodash';
+import { getKeyByValue } from 'utils/generic-functions';
+
+import { getTerrestrialCellData } from 'selectors/grid-cell-selectors';
+
+export const COMMUNITY_BASED = 'community';
+export const PROTECTED = 'protected';
+export const NOT_UNDER_CONSERVATION = 'notUnderConservation';
+
+const COLORS = () => ({
+  [COMMUNITY_BASED]: '#FCC44A',
+  [PROTECTED]: '#FF6C47',
+  [NOT_UNDER_CONSERVATION]: '#6C828F'
+})
+
+const conservationEffortsData = ({ conservationEffortsData }) => conservationEffortsData && conservationEffortsData.data;
+
+const getConservationEfforts = createSelector(
+  [conservationEffortsData],
+  cellData => {
+    if (!cellData) return null;
+    const conservationEfforts = cellData.reduce((acc, current) => {
+      return {
+        ...acc,
+        [current.CELL_ID]: {
+          WDPA_prop: current.WDPA_prop,
+          RAISG_prop: current.RAISG_prop,
+          all_prop: current.all_prop
+        }
+        }
+    }, {});
+    const values = Object.values(conservationEfforts)
+    const gridCellsLength = Object.keys(conservationEfforts).length;
+    const WDPA_prop = sumBy(values, 'WDPA_prop') / gridCellsLength;
+    const RAISG_prop = sumBy(values, 'RAISG_prop') / gridCellsLength;
+    const all_prop = sumBy(values, 'all_prop') / gridCellsLength;
+    return {
+      WDPA_prop,
+      RAISG_prop,
+      all_prop
+    }
+  }
+)
+
+const getConservationAreasLogic = createSelector(
+  [getConservationEfforts],
+  (conservationEfforts) => {
+    if (!conservationEfforts) return null;
+
+    const areas = {};
+    if (conservationEfforts.WDPA_prop + conservationEfforts.RAISG_prop > conservationEfforts.all_prop) {
```

Comment:
Here I'd recommend using destructuring, just to increase readability of this condition :) :
`const { WDPA_prop, RAISG_prop, all_prop } = conservationEfforts;`